builtin/mainmenu/tab_content.lua

```diff
@@ -0,0 +1,216 @@
+
+		if selected_pkg.type == "mod" then
+			if selected_pkg.is_modpack then
+				retval = retval .. ";0]" ..
+					"button[8.9,4.65;3,1;btn_mod_mgr_rename_modpack;" ..
+					fgettext("Rename") .. "]"
+			else
+				--show dependencies
+				local toadd_hard = table.concat(info.depends or {}, ",")
+				local toadd_soft = table.concat(info.optional_depends or {}, ",")
+				if toadd_hard == "" and toadd_soft == "" then
+					retval = retval .. "," .. fgettext("No dependencies.")
+				else
+					if toadd_hard ~= "" then
+						retval = retval .. "," .. fgettext("Dependencies:") .. ","
+						retval = retval .. toadd_hard
+					end
+					if toadd_soft ~= "" then
+						if toadd_hard ~= "" then
+							retval = retval .. ","
+						end
+						retval = retval .. "," .. fgettext("Optional dependencies:") .. ","
+						retval = retval .. toadd_soft
+					end
+				end
+
+				retval = retval .. ";0]"
+			end
+
+		else
+			retval = retval .. ";0]"
+
+			if selected_pkg.type == "txp" then
+				if selected_pkg.enabled then
+					retval = retval ..
+						"button[8.9,4.65;3,1;btn_mod_mgr_disable_txp;" ..
+						fgettext("Disable Texture Pack") .. "]"
+				else
+					retval = retval ..
+						"button[8.9,4.65;3,1;btn_mod_mgr_use_txp;" ..
+						fgettext("Use Texture Pack") .. "]"
+				end
+			end
+		end
+
+		retval = retval .. "button[5.5,4.65;3,1;btn_mod_mgr_delete_mod;"
+			.. fgettext("Uninstall Package") .. "]"
+	end
+	return retval
+end
+
+--------------------------------------------------------------------------------
+local function handle_buttons(tabview, fields, tabname, tabdata)
+	if fields["pkglist"] ~= nil then
+		local event = core.explode_table_event(fields["pkglist"])
+		tabdata.selected_pkg = event.row
+		return true
+	end
+
+	if fields["btn_mod_mgr_install_local"] ~= nil then
+		core.show_file_open_dialog("mod_mgt_open_dlg", fgettext("Select Package File:"))
+		return true
+	end
+
+	if fields["btn_contentdb"] ~= nil then
+		local dlg = create_store_dlg()
+		dlg:set_parent(tabview)
+		tabview:hide()
+		dlg:show()
+		packages = nil
+		return true
+	end
+
+	if fields["btn_mod_mgr_rename_modpack"] ~= nil then
+		local dlg_renamemp = create_rename_modpack_dlg(tabdata.selected_pkg)
+		dlg_renamemp:set_parent(tabview)
+		tabview:hide()
+		dlg_renamemp:show()
+		return true
+	end
+
+	if fields["btn_mod_mgr_delete_mod"] ~= nil then
+		local mod = packages:get_list()[tabdata.selected_pkg]
+		local dlg_delmod = create_delete_content_dlg(mod)
+		dlg_delmod:set_parent(tabview)
+		tabview:hide()
+		dlg_delmod:show()
+		packages = nil
+		return true
+	end
+
+	if fields.btn_mod_mgr_use_txp then
+		local txp = packages:get_list()[tabdata.selected_pkg]
+		core.settings:set("texture_path", txp.path)
+		packages = nil
+		return true
+	end
+
+
+	if fields.btn_mod_mgr_disable_txp then
+		core.settings:set("texture_path", "")
+		packages = nil
+		return true
+	end
+
+	if fields["mod_mgt_open_dlg_accepted"] ~= nil and
+		fields["mod_mgt_open_dlg_accepted"] ~= "" then
```

Comment:
Missing indent, in this case redundant `~= nil`
Line below: missing space after `,`